cli/command/formatter/service.go

```diff
@@ -19,9 +19,9 @@ Labels:
 {{- range $k, $v := .Labels }}
  {{ $k }}{{if $v }}={{ $v }}{{ end }}
 {{- end }}{{ end }}
-Mode:
-{{- if .IsModeGlobal }}		Global
-{{- else }}		Replicated
+Service Mode:
+{{- if .IsModeGlobal }}	Global
+{{- else }}	Replicated
```

Comment:
@aaronlehmann 
Thanks a lot for review.
I changed `Mode` into `Service Mode`, so I think there is only one tab needed.
PTAL.